config.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 # Location of database
-DATABASE_FILE_PATH = os.path.abspath(os.getenv('DATABASE_FILE_PATH', 'data.db'))
+DATABASE_FILE_PATH = os.path.abspath(os.getenv('DATABASE_FILE_PATH', 'data/data.db'))
```

Comment:
The default needs to be left as `'data.db'` to not make any breaking changes. You can override the value using an environment variable.